### PR TITLE
review-submission-update

### PR DESCRIPTION
**Pull request by nimithajalal**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/guides/review/index.md [hunk 1]
```diff
@@ -6,17 +6,39 @@ title: Submission and Review
...
-# Submission and Review
+## Overview
 
-Adobe reviews all applications that integrate Adobe Express Embed SDK before enabling it for public use.
+This guideline outlines the process for reviewing and submitting an integration using the Adobe Express Embed SDK. Whether you’re embedding the Full Editor, Quick Actions, or Modules, adhering to these guidelines will ensure a seamless experience for users and maintain consistency across all implementations.
 
-## Before you submit
+<InlineAlert slots="text" />
+
+We aim to review your integration within 10 business days of submission, and will let you know if it is accepted, or if any changes need to be made.
```
Comment (undavide) [inline src/pages/guides/review/index.md:18]: I'd use bold if we are confident that the review team can make it in 10 days :-) 

We aim to review your integration **within 10 business days of submission**, and will let you know if it is accepted, or if any changes need to be made.

File: src/pages/guides/review/index.md [hunk 2]
```diff
@@ -35,28 +57,84 @@ By creating a project using the SDK in Adobe Developer Console, you are agreeing
 
 Additionally, your integration must not violate the trademarks and copyrights of Adobe, Inc., or those of any other party. Please adhere to all trademark guidelines as noted here: <https://www.adobe.com/legal/permissions/trademarks.html>
```
Comment (undavide) [inline src/pages/guides/review/index.md:58]: I'd remove the explicit link:

❌ Please adhere to all trademark guidelines as noted here: <https://www.adobe.com/legal/permissions/trademarks.html>

✅ Please adhere to all trademark guidelines as noted [here](https://www.adobe.com/legal/permissions/trademarks.html).

File: src/pages/guides/review/index.md [hunk 3]
```diff
@@ -35,28 +57,84 @@ By creating a project using the SDK in Adobe Developer Console, you are agreeing
...
 
-## Submit your app
+## Submission Guidelines
+
+To make sure your review process goes smoothly, check off the tasks in this list before submitting.
+
+If you have any questions, feel free to contact us at [ccintrev@adobe.com](mailto:ccintrev@adobe.com) or reach out on our [Adobe Express Embed community forum](https://community.adobe.com/t5/express-embed-sdk/ct-p/ct-express-embed-sdk?page=1&sort=latest_replies&lang=all&tabid=all).
```
Comment (undavide) [inline src/pages/guides/review/index.md:64]: Add SDK:

reach out on our [Adobe Express Embed 👉 SDK 👈 community forum](https://community.adobe.com/t5/express-embed-sdk/ct-p/ct-express-embed-sdk?page=1&sort=latest_replies&lang=all&tabid=all).

File: src/pages/guides/review/index.md [hunk 4]
```diff
@@ -35,28 +57,84 @@ By creating a project using the SDK in Adobe Developer Console, you are agreeing
...
+
+<InlineAlert variant="help" slots="text" />
+
+- Confirm that the `appName` used to initialize the SDK in your application code reflects your app's name. 
+- Note any projects created through your integration will be saved in the user's Express account, inside an Express folder with this name.
+- Confirm that the SDK is never initialized more than once per page load.
+
+### 1. Submit for approval
+
+Navigate to your project in Adobe Developer Console -- select **Submit for review** from the __Project overview__ tab or select the __Approval__ tab on the left panel.
```
Comment (undavide) [inline src/pages/guides/review/index.md:74]: Nit: is the `--` intended as an em dash? Maybe:  
... in Adobe Developer Console; select...
... in Adobe Developer Console—select...
(totally marginal comment)

File: src/pages/guides/review/index.md [hunk 5]
```diff
@@ -35,28 +57,84 @@ By creating a project using the SDK in Adobe Developer Console, you are agreeing
...
-   * Please list which quick actions(s), if any, you use in your application.
-   * Please list which modules, if any, you use in your application.
-   * Please list the intended behavior of any custom export buttons you use with the full editor, modules, or quick actions.
-   * Confirm that the `appName` used to initialize the SDK in your application code reflects your app's name. Note any projects created through your integration will be saved in the user's Express account, inside an Express folder with this name.
-   * Confirm that the SDK is never initialized more than once per page load.
+IMPORTANT: Deprecation Warning
 
-5. Complete the __Screenshots for approval__ section.
-6. Once you think you're good to go, click the "Submit for Review" button. Our review team will ensure your integration fulfills all the guidelines and passes all the [test cases](#test-cases) before approving it for public usage.
+We are no longer approving integrations using v1 or v2 of the SDK - both versions will be deprecated in 2024.
```
Comment (undavide) [inline src/pages/guides/review/index.md:98]: We know these will change, as v1 + v2 are already deprecated, and v3 is not accepted anymore.
Should we say that v3 is going to be deprecated in 2025 or just leave it be?

Perhaps this alert box could be moved just above the Notes to reviewer?

File: src/pages/guides/review/index.md [hunk 6]
```diff
@@ -35,28 +57,84 @@ By creating a project using the SDK in Adobe Developer Console, you are agreeing
...
+If your app processes personal information, include a link to the privacy policy on your app’s website, per the Adobe Developer Agreement.
+
+### 4. Details for Approval
+
+Describe what your app does, to help Adobe review it for approval. Allowing other enterprises to discover your app or integration in the Adobe Admin Console will make it so that the administrators of those enterprises can manage access to it for their users. 
 
-1. Navigate to your project in Adobe Developer Console -- select "Submit for review" from the __Project overview__ tab or select the __Approval__ tab on the left panel.
-   ![Submit for review in Console UI](submission.png)
-2. If this is your first Adobe integration, set up your public profile - note that the information that appears on your public profile will appear in the consent screen displayed to users.
+### 4. Notes to reviewer
```
Comment (undavide) [inline src/pages/guides/review/index.md:92]: This should be 5

File: src/pages/guides/review/index.md [hunk 7]
```diff
@@ -35,28 +57,84 @@ By creating a project using the SDK in Adobe Developer Console, you are agreeing
...
-6. Once you think you're good to go, click the "Submit for Review" button. Our review team will ensure your integration fulfills all the guidelines and passes all the [test cases](#test-cases) before approving it for public usage.
+We are no longer approving integrations using v1 or v2 of the SDK - both versions will be deprecated in 2024.
 
-### Test cases
+Make sure you’ve included the following in the **Notes to reviewer** feild.
+
+1. **Test credentials** for your integration. 
+2. **Workflow** of your integration within your product (URL, step-by-step instructions, and more.)
+3. **Version** of the SDK you are using.
+4. List the intended behavior of any custom export buttons you use with the full editor, modules, or quick actions.
```
Comment (undavide) [inline src/pages/guides/review/index.md:105]: Maybe add a menacing statement after the list, like: Failing to provide these details will sadly result in an automatic rejection, and you'll be shamed by your friends and family forever. Another alert box could be useful to draw attention.

File: src/pages/guides/review/index.md [hunk 8]
```diff
@@ -35,28 +57,84 @@ By creating a project using the SDK in Adobe Developer Console, you are agreeing
...
+
+### 5. Provide screenshots
```
Comment (undavide) [inline src/pages/guides/review/index.md:107]: This should be 6

File: src/pages/guides/review/index.md [hunk 9]
```diff
@@ -76,20 +153,17 @@ Additionally, your integration must not violate the trademarks and copyrights of
...
-
-Once your submission has been reviewed, we will email you letting you know whether your submission was approved or rejected. If it was rejected, we will include the reasons for rejection along with any next steps. If you'd like to back and edit your submission, you can do so in the Developer Console.
+-  Your app or integration is not accessible via free user sign-up and test credentials are not provided to our review team for testing purposes.
+- Your app description is not complete in the __Notes to reviewer__ section, and our review team cannot test all the entrypoints. Make sure you are clear about what your app integrates (full editor, modules, and/or quick actions) and where/how to invoke them.
+- Your app name should correspond with the name of your project in Adobe Developer Console. This `appName` should also be consistent throughout your application. We use this property in our analytics checks so if there are any discrepancies, we will ask that you make that change before you are able to release the integration to your users.
```
Comment (undavide) [inline src/pages/guides/review/index.md:160]: I'd sprinkle some bold to make them pop:

-  Your app or integration is **not accessible via free user sign-up** and **test credentials are not provided** to our review team for testing purposes.
- Your app **description is not complete** in the __Notes to reviewer__ section, and our review team **cannot test all the entrypoints**. Make sure you are clear about what your app integrates (full editor, modules, and/or quick actions) and where/how to invoke them.
- Your **app name should correspond with the name of your project** in Adobe Developer Console. This `appName` should also be consistent throughout your application. We use this property in our analytics checks so if there are any discrepancies, we will ask that you make that change before you are able to release the integration to your users.

**Review by undavide — APPROVED** [review]
Great improvements!! 
I have just a few minor comments, with the exception of the 4. which appears twice in the ordered list.

File: src/pages/guides/review/index.md (hunk 1 above)
Comment (nimithajalal) [reply src/pages/guides/review/index.md:22]: We are not really confident here.

**Pull request merged**